.gitmodules

```diff
@@ -1,6 +1,30 @@
 [submodule "backup"]
 	path = backup
 	url = https://github.com/vimc/montagu-backup
-[submodule "database"]
-	path = database
-	url = https://github.com/vimc/montagu-db.git
```

Comment:
We added this database submodule at one point when we were running migrations manually.